pom.xml

```diff
@@ -89,23 +89,23 @@
         <maven.templating.version>1.0.0</maven.templating.version>
 
         <!-- dependencies -->
-        <jfx.version>18</jfx.version>
-        <kotlin.version>1.6.0</kotlin.version>
+        <jfx.version>18.0.2</jfx.version>
+        <kotlin.version>1.7.10</kotlin.version>
```

Comment:
There are changes to multiple aspects of this file. Could we minimise the changes just to upgrading the kotlin and javafx version?